controllers/frame_controller.py

```diff
@@ -0,0 +1,19 @@
+from flask import request
+from draft.BionicEye.singelton_classes.db_manager import DBManager
+from draft.BionicEye.singelton_classes.os_manager import OSManager
+from draft.BionicEye.models import Frame
+
+DB_MANAGER = DBManager()
+OS_MANAGER = OSManager()
+
+
+def get_frame():
+    """
+    Gets from the db the frame path with the index and video id given in the request
+    :return: path of the frame in the os
+    """
+    video_id = request.args.get("video_id")
+    frame_index = request.args.get("frame_index")
```

Comment:
Change this function to be more generic and get the parameters in the routes.py and send them to this function.
What do you say ?